GenPackageDoc/styles/robotframeworkaio.sty

```diff
@@ -267,7 +270,7 @@
       keywordstyle=\color{darkblue},   
       morekeywords=[1]{PASS, passed},
       morekeywords=[2]{FAIL, failed},
-      keywordstyle=[1]\color{green},
+      keywordstyle=[1]\color{ForestGreen},
```

Comment:
Of course "ForestGreen". This is what I like to have around me the whole day :-)